update/ansible/playbook/tasks/roles/nginx/files/conf.d/pmm.conf

```diff
@@ -239,6 +239,12 @@
       proxy_set_header Connection "";
     }
 
+    # pmm-dump artifacts
+    location  /dump/ {
+        root /srv;
+        autoindex on;
+    }
+
```

Comment:
Download link example: http://localhost/dump/f835fe8b-b323-47a4-9623-049a8471c9e9.tar.gz
Also directory content available at: http://localhost/dump/

Reply:
let's put in subfolder for better secutiry

Reply:
`/dump/` form URL used as part of final destination, i.e. other folders in `/srv` are not accessible.

Reply:
BTW I'm not really good in `nginx` configuration, so I you know how to improve I will appreciate

Reply:
it seems like `alias` would work better here to not create confusions https://fedingo.com/nginx-alias-vs-root/.
Do we really need autoindex to be enabled?

Reply:
It's nice to have this view, I guess:
![Uploading Screenshot 2023-11-02 at 15.28.28.png…]()

Reply:
we have our UI for that, so I think we don't need that.

Reply:
Our UI is for user, and this one more for developers (may be useful for debugging). But ok, will remove it.

Reply:
Done